timeout_config.hh

```diff
@@ -10,20 +10,60 @@
 #pragma once
 
 #include "db/timeout_clock.hh"
+#include "utils/updateable_value.hh"
+#include <chrono>
+
+namespace db { class config; }
+
+class updated_timeout_config;
 
 struct timeout_config {
-    db::timeout_clock::duration read_timeout;
-    db::timeout_clock::duration write_timeout;
-    db::timeout_clock::duration range_read_timeout;
-    db::timeout_clock::duration counter_write_timeout;
-    db::timeout_clock::duration truncate_timeout;
-    db::timeout_clock::duration cas_timeout;
-    db::timeout_clock::duration other_timeout;
+    using duration_t = db::timeout_clock::duration;
+
+    static timeout_config from_updated(const updated_timeout_config&);
```

Comment:
Unused

Reply:
removed. it was an leftover from previous refactory.